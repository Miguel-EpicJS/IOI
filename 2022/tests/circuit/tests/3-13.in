385045b28fbc1903eca997d513040235e0229e01
499 999 5
-1 0 1 0 1 3 3 1 6 6 8 0 8 12 5 9 3 6 11 16 12 19 10 5 18 14 8 20 25 14 23 4 30 29 31 24 33 12 31 34 35 29 14 28 15 29 38 11 33 25 47 43 42 5 45 48 30 24 56 23 31 55 42 49 53 16 60 65 37 49 65 25 67 52 56 70 48 47 70 59 47 42 75 73 45 70 84 77 62 60 87 56 71 59 37 87 64 19 32 97 71 76 97 81 100 89 100 103 97 89 108 81 74 54 86 89 104 81 45 52 61 2 73 30 118 59 32 19 18 118 110 33 94 128 17 72 16 128 117 117 113 132 106 135 119 142 93 119 80 67 80 132 35 83 53 133 112 147 137 155 146 108 109 155 132 35 138 144 117 167 158 133 84 170 87 161 127 123 114 129 50 136 102 167 145 163 137 172 183 183 144 171 160 102 131 134 152 129 122 169 67 106 198 139 76 142 200 186 200 158 192 72 134 191 112 210 110 215 201 208 176 161 172 177 197 145 37 197 163 205 175 210 172 225 185 176 86 206 73 235 108 167 160 200 150 208 68 244 157 194 240 113 219 193 248 145 209 237 124 251 127 192 177 149 233 72 263 170 250 240 259 144 232 242 255 155 124 229 177 215 260 207 254 113 216 147 265 207 245 238 210 160 260 260 156 248 158 264 205 118 71 262 280 241 289 295 146 283 273 171 306 288 238 292 250 299 287 299 236 306 205 319 168 304 309 186 299 121 262 291 176 319 83 293 326 306 182 146 314 226 119 322 256 337 288 175 149 88 238 286 100 125 301 295 319 94 251 335 318 243 84 261 293 154 350 169 248 269 221 362 268 362 367 321 347 371 360 293 295 209 221 272 137 378 367 371 224 371 52 382 377 192 251 338 291 358 106 390 389 365 215 337 388 393 401 396 367 360 268 263 394 390 142 361 409 338 300 335 402 298 395 395 396 250 360 370 302 418 378 243 298 354 378 427 406 398 149 402 388 419 318 49 384 276 423 433 398 440 393 340 379 427 438 432 362 408 376 456 390 441 446 450 436 412 456 361 434 320 401 434 440 458 469 340 472 397 462 449 392 472 474 445 478 458 476 469 458 463 449 469 467 486 474 474 427 472 484 449 488 2 4 2 9 7 7 10 7 9 13 13 23 22 24 20 11 15 22 20 26 27 17 15 18 36 4 26 44 39 43 17 28 40 13 39 27 27 44 48 38 57 51 44 10 46 55 43 21 53 21 34 65 40 38 28 41 58 60 26 75 66 46 75 77 64 40 41 64 69 58 74 79 51 39 79 78 46 82 90 68 76 80 83 34 77 88 82 92 55 69 101 32 96 90 85 82 50 90 36 103 109 63 86 21 111 79 85 95 101 74 104 91 51 114 94 91 98 41 63 93 66 116 50 92 130 107 125 95 95 126 114 126 136 88 122 130 78 92 111 96 116 63 103 139 99 98 115 121 62 54 153 93 110 151 123 111 101 151 152 133 58 54 135 164 162 131 98 138 159 121 157 150 156 166 141 169 148 134 120 85 138 164 130 166 135 96 161 153 123 107 105 187 122 189 184 194 115 152 162 116 143 170 57 181 120 180 78 204 156 207 188 62 188 209 168 148 105 129 61 151 168 179 213 140 184 125 181 174 199 139 190 136 153 175 211 203 195 36 154 105 165 217 140 219 196 214 182 143 211 107 195 191 187 216 220 115 102 191 179 233 157 189 231 231 222 69 196 225 220 183 214 120 193 91 165 173 61 181 223 162 230 187 222 178 202 212 124 194 159 229 189 218 99 109 195 228 230 203 143 198 150 203 233 223 218 239 179 178 212 235 245 258 188 164 254 197 256 199 127 226 171 235 259 246 263 252 128 236 272 22 269 202 234 225 249 126 278 224 190 174 272 274 239 258 242 222 253 214 57 201 282 131 266 286 217 68 267 253 270 208 206 297 246 173 296 277 280 265 204 282 265 232 297 241 274 249 285 283 283 301 244 278 308 261 217 249 294 257 240 277 173 278 178 305 165 284 284 246 228 296 211 196 310 303 271 310 314 104 317 221 159 199 270 321 315 202 218 154 234 307 174 281 276 327 328 302 294 314 317 256 324 268 286 279 273 184 236 185 224 206 228 322 273 320 261 326 292 332 312 266 311 305 308 227 323 264 287 231 147 311 193 300 141 281 311 294 343 252 244 307 266 315 341 247 324 322 332 239 258 315 330 342 290 338 190 349 243 274 343 201 229 280 281 182 204 245 312 316 275 303 316 336 334 351 316 227 325 353 254 255 323 327 230 358 185 300 252 334 241 304 270 331 112 333 346 237 313 288 296 290 276 163 365 363 242 350 277 219 329 339 354 363 237 271 344 332 227 357 275 186 366 284 259 307 267 341 347 287 257 345 289 366 335 333 321 324 368 297 337 247 325 291 212 368 346 375 303 359 356 356 301 329 364 326 308 348 379 346 310 356 349 334 309 289 318 269 323 380 275 312 328 336 342 375 352 141 166 372 327 330 354 365 220 344 385 385 232 359 381 369 285 285 253 386 364 247 374 386 180 359 357 320 180 376 305 347 257 351 329 361 290 66 394 309 358 226 350 384 339 379 376 396 325 393 372 313 372 384 353 348 262 381 366 148 382 345 317 401 395 388 330 198 373 405 298 374 399 387 389 331 397 357 302 383 355 374 405 364 140 402 375 343 216 292 406 400 405 415 400 279 417 409 279 383 412 380 411 404 407 416 413 419 416 271 331 223 408 349 406 377 418 417 267 353 352 328 348 392 382 413 412 344 342 414 425 234 418 413 422 340 428 368 408 391 429 426 336 213 421 403 420 420 380 425 411 404 264 424 387 370 398 410 407 351 414 407 429 436 420 422 369 397 434 428 381 399 433 333 430 394 441 437 377 439 410 213 424 438 399 373 352 446 387 428 435 448 391 430 400 416 451 389 415 304 421 442 447 439 419 424 414 392 423 370 443 445 430 425 451 433 341 363 422 373 403 445 444 456 450 459 443 442 440 282 99 435 255 466 443 355 437 391 457 459 431 313 465 455 452 465 452 415 455 465 461 403 450 464 438 444 468 460 429 404 471 454 431 457 478 461 479 369 339 475 480 473 355 447 441 479 423 448 385 462 417 454 435 476 464 467 442 466 453 479 484 457 477 466 461 470 482 476 475 467 483 452 410 436 454 460 464 455 451 482 462 463 446 383 486 471 421 477 447 453 487 444 471 463 468 386 345 426 411 485 439 432 488 488 437 487 470 481 491 487 489 460 483 473 473 489 490 492 489 485 483 494 492 459 494 477 426 495 468 490 481 495 493 453 448 496 493 478 432 492 482 497 484 475 470 486 490 431 480 491 497 409 491 481 480 496 496 495 485 497 494 498 498 498 493
1 1 1 1 1 0 1 1 0 0 1 1 0 1 1 1 0 1 0 0 1 1 0 0 1 0 0 0 1 0 1 1 0 0 0 1 1 0 1 1 0 0 0 1 0 0 0 1 1 1 0 0 0 1 1 1 0 1 0 0 1 1 1 1 0 1 1 1 0 1 0 0 0 1 1 1 1 0 1 1 0 1 1 1 0 0 0 1 1 0 1 1 0 0 0 1 0 1 0 1 1 0 1 1 1 0 0 0 1 1 0 0 0 0 1 1 1 1 1 1 0 1 0 1 1 0 1 0 1 0 1 1 1 0 0 1 0 0 0 0 0 0 0 1 0 1 1 1 1 1 1 1 1 0 0 0 0 0 0 1 1 1 0 0 1 0 1 1 1 0 1 1 1 1 1 1 1 1 0 0 1 0 0 1 1 0 0 0 1 0 1 0 0 0 1 0 1 1 0 1 0 1 1 1 1 0 1 0 0 1 1 1 0 0 1 0 1 1 1 1 0 1 0 0 0 1 1 0 1 0 1 0 1 1 0 0 0 0 0 0 0 0 1 1 1 1 1 1 1 1 0 1 0 1 1 0 0 0 0 1 0 1 1 0 0 0 1 0 1 1 1 0 0 1 1 0 1 1 0 1 1 1 1 0 1 1 0 0 1 0 0 0 0 0 0 1 1 0 1 1 1 0 1 0 1 1 1 0 1 0 1 1 0 0 0 1 1 1 1 0 1 1 0 1 0 1 0 0 0 1 1 0 1 1 1 0 1 1 0 0 1 1 0 1 1 1 1 1 0 1 1 0 1 1 1 0 0 0 0 1 1 0 1 0 1 1 1 0 0 1 0 0 0 1 0 1 1 0 0 0 1 1 0 1 1 0 1 1 0 0 0 0 0 1 0 0 0 1 0 0 1 0 1 0 1 1 1 1 0 0 0 0 0 1 0 0 0 1 1 0 0 0 1 1 0 1 0 1 1 0 1 1 1 1 1 1 1 1 0 0 1 0 0 1 1 1 0 0 1 1 1 1 1 0 1 1 1 0 1 1 0 1 0 0 1 0 1 0 0 1 1 0 1 1 0 0 0 1 0 0 1 0 0 0 1 1 0 1 1 0 1 0 0 0 1 1 1 0 1 0 0 1 1 0 0 1 1 1 1 0 0 1 1 0 1 0 1 0 0 1 1 0 0 1 1 0 1 1 1 0 0 1 1 0 0 1 0 0 0 1 1 0 0 1 0 1 0 0 0 0 1 0 0 0 0 0 1 1 0 0 0 0 0 0 1 1 1 0 0 1 0 1 1 0 0 0 1 1 1 1 0 1 1 1 0 1 1 0 0 1 1 1 1 0 1 1 0 0 0 0 1 1 1 1 0 1 1 1 1 1 0 1 1 1 0 0 1 0 1 1 0 0 0 0 1 1 0 0 0 0 0 0 1 0 0 1 1 0 0 0 1 1 1 1 1 1 0 0 0 0 0 1 0 0 0 1 0 1 0 0 0 0 0 1 1 1 1 0 0 0 1 1 1 1 1 0 0 1 0 0 1 0 1 0 0 0 0 0 1 0 0 0 0 0 0 1 0 0 1 1 0 0 0 1 0 0 1 0 1 1 1 1 0 1 1 1 0 0 1 0 0 1 1 1 0 0 0 0 0 0 1 1 0 1 1 0 0 0 0 0 1 1 1 1 1 1 1 0 0 0 1 1 1 1 0 1 1 0 0 0 1 1 0 0 0 0 1 1 1 0 1 1 0 0 1 1 1 0 1 0 0 1 1 1 1 0 1 1 1 0 0 1 0 1 1 1 1 1 1 0 0 0 1 0 0 1 0 1 1 0 0 0 1 0 0 1 0 1 1 0 0 0 0 0 0 0 0 1 0 0 1 0 1 0 0 1 0 1 0 1 0 0 1 1 1 0 1 0 1 0 1 0 1 0 0 0 0 1 1 1 0 1 1 0 0 1 0 0 0 1 1 0 1 1 1 1 0 0 0 1 1 1 0 1 1 0 0 1 0 1 0 1 0 1 1 1 0 0 1 1 0 1 1 0 1 1 0 0 0 0 0 1 1 0 0 1 0 1 0 1 0 1 0 1 0 0 0 1 0 1 0 1 1 0 0 0 1 1 1 1 1 0 1 0 0 0 1 0 1 1 1 0 1 0 1 1 1 1 1 0 1 0 0 1 1 0 0 0 0 1 0 1 0 0 1 1 0 1 0 1 1 1 1 0 1 0 1 1 0 1 0 1 0 1 0 1 0 0 1
1061 1408
513 908
663 1468
907 1294
730 1317
