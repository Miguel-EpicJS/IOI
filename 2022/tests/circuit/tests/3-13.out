4faa78c22ee35083e49b65601698a1bd98a87a0f
OK
331823726
717282174
897711128
622871150
337192684
